browse-handler/java/org/vufind/solr/handler/client/solrj/BrowseRequest.java

```diff
@@ -0,0 +1,53 @@
+package org.vufind.solr.handler.client.solrj;
+
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrRequest;
+import org.apache.solr.common.params.SolrParams;
+
+public class BrowseRequest extends SolrRequest<BrowseResponse>
```

Comment:
Should we add a class-level comment explaining why this is here? If it's only used for testing purposes at this time, that might be worth mentioning.